server/controllers/appointmentController.ts

```diff
@@ -424,15 +410,15 @@ export default class AppointmentController {
             ])
 
             return {
-              bookingType: !bvlsMasteredVlpmFeatureToggleEnabled() ? vlb.bookingType : undefined,
+              bookingType: undefined as string,
```

Comment:
Not sure why we wouldn't put the bookingType here? But agree that this is what the code was doing when the toggle is unset.

Reply:
Unless this is a court booking!

Reply:
Which it is, of course.

Reply:
Agree, though it does look/read strange.